envars/envars.py

```diff
@@ -215,7 +215,8 @@ def execute(args):
         args.var = None
         ret = process(args)
         for val in ret:
-            vals[val.split('=')[0]] = val.split('=')[1]
+            matches = re.match(r'^([A-Z][A-Z|0-9|_]+)=(.*)$', val)
```

Comment:
Do you mean it would split on multiple `=` 

Say you had 

`KEY=Secret=pass` 

you can do  

```key, val = vars.split("=",1)``` use the one to limit the number of splits

and key would be KEY and val would be Secret=pass

Reply:
ah I did not know that!